frontend/packages/console-shared/src/sorts/nodes.ts

```diff
@@ -0,0 +1,15 @@
+import * as UIActions from '@console/internal/actions/ui';
```

Comment:
Pondering this new location for `sorts`, given we also have a way to pass custom sorts https://github.com/openshift/console/pull/4708/files#diff-bc19a445cd55d99705e404fae2c786edR338

Reply:
Scratch that, I see its consistent with what's already in `sorts` for pods.